docs/Views/main.js

```diff
@@ -0,0 +1,36 @@
+import loginFunctions from '../firebase/login.js';
+
+export default () => {
+  const loginView = document.createElement('div');
+  loginView.setAttribute('class', 'viewsStyle');
+  loginView.innerHTML = ` 
+            <center><h3>Inicia Sesión con tu email</h3></center>
```

Comment:
Center no es la mejor manera de centrar los elementos, la recomendación es centrar por medio de css.
```suggestion
            <h3 class="h3-center">Inicia Sesión con tu email</h3>
```
pueden utilizar  justify-content: center